### PR TITLE
Add hotReloadProfile to Razor Pages template

### PR DESCRIPTION
**Pull request by pranavkm**
Porting https://github.com/dotnet/aspnetcore/pull/31197 to preview3

### CODE REVIEW

File: src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Properties/launchSettings.json [hunk 1]
```diff
@@ -29,7 +29,6 @@
       "commandName": "Project",
       "dotnetRunMessages": "true",
       "launchBrowser": true,
-      "hotReloadProfile": "blazorwasm",
```
Comment (pranavkm) [inline src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Properties/launchSettings.json:32]: FYI @SteveSandersonMS / @danroth27

Comment (danroth27) [reply src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Properties/launchSettings.json:32]: Makes sense to hold off on enabling hot reload for Blazor WebAssembly by default until the experience is a bit more baked. Users can opt into trying it out if they want.

**Review by TanayParikh — APPROVED** [review]

**Review by danroth27 — APPROVED** [review]

File: src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Properties/launchSettings.json (hunk 1 above)
Comment (lambdageek) [reply src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Properties/launchSettings.json:32]: Hold off on merging this.  I'm working on the bug

Comment (lambdageek) [reply src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Properties/launchSettings.json:32]: https://github.com/dotnet/runtime/pull/50248 for main. I'll also open a backport for preview3

**Pull request merged**